packages/tools/lint-package/src/helpers.ts

```diff
@@ -0,0 +1,36 @@
+import type { Rules } from './types.js';
+
+import JSONC from 'jsonc-parser';
+import fs from 'node:fs/promises';
+import { dirname, extname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const DIRNAME = dirname(fileURLToPath(import.meta.url));
+const RULES_DIR = join(DIRNAME, 'rules');
+
+async function importModules(dir: string): Promise<Rules> {
+  const files = await fs.readdir(dir);
+  const moduleFiles = files.filter((file) => extname(file) === '.js');
+  const modules = [];
+  for (const file of moduleFiles) {
+    const module = await import(new URL(join(dir, file), 'file://').toString());
+    modules.push(module.default);
+  }
+  return modules;
+}
+
+export const loadJSON = async <T>(filePath: string): Promise<T> =>
+  JSON.parse(String(await fs.readFile(filePath)));
+
+export const loadJSONC = async <T>(filePath: string): Promise<T> =>
+  JSONC.parse(String(await fs.readFile(filePath)));
+
+export const loadContents = async (filePath: string) =>
+  String(await fs.readFile(filePath));
```

Comment:
We can remove the need for `String()` by adding the encoding

```suggestion
export const loadJSON = async <T>(filePath: string): Promise<T> =>
  JSON.parse(await fs.readFile(filePath, 'utf-8'));

export const loadJSONC = async <T>(filePath: string): Promise<T> =>
  JSONC.parse(await fs.readFile(filePath, 'utf-8'));

export const loadContents = async (filePath: string): Promise<string> =>
 await fs.readFile(filePath, 'utf-8');
```

Reply:
Ah of course, thanks